### DISCUSSION TITLE
How do I secure my Github account?

### DISCUSSION THREAD

**Discussion by localhost-labs** [community/community#66071] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm new to Github and don't want to put anything in here until it is secure

**Comment by chrisreeves-** [comment]
https://docs.github.com/en/authentication/keeping-your-account-and-data-secure

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
